### ISSUE TITLE
rename "posts" to include course names

### ISSUE THREAD

**Issue by LauraHilliger** [mozilla/school-of-webmaking#113]
Currently posts are 2000-01-02, which isn't very intuitive. I'm wondering where this is set, or if I can just change the names to like "exploring-resources.markdown" without breaking everything? It's not that big of a deal now, but as we do the SoW audit, I feel like I need to understand how to add new stuff :)

**Comment by dirkcuys** [comment]
Unfortunately that is a quirk from the way we use Jekyll :( Course pages are created as posts and posts need to include the date, we are stuck with this.

The alternative is to move the course pages to ex. `/exploring/resources.markdown`. This was my initial preference, but Jekyll doesn't treat pages and posts the same and listing and ordering pages isn't the same way as with posts.

There are alternatives like not using Jekyll, which can work for us, but would mean anyone without the know how and an Amazon S3 account won't be able to easily fork a course.

**Comment by LauraHilliger** [comment]
Not quite sure, but it seems like you're saying we can't organize them into subfolders or to get around this? Possibility of writing a patch to Jekyll (??)

yes, we need to keep the forking thing easy peasy. Hrm...

**Comment by dirkcuys** [comment]
Problem with writing a plugin for Jekyll is that github doesn't actually allow you to use any plugins when using github pages :(

Wrt adding new stuff like a new module, I wrote some basic docs here: https://github.com/p2pu/school-of-webmaking/blob/gh-pages/docs/adding-a-course.markdown

**Comment by fuzzyfox** [comment]
We might be able to do something about this now that [GH uses Jekyll 2](https://github.com/blog/1867-github-pages-now-runs-jekyll-2-2-0) (and thus [collections](http://jekyllrb.com/docs/collections/) other than posts)

**Comment by dirkcuys** [comment]
Yes, I want to investigate this. The question that still remains for me that the Jekyll documentation doesn't address is how does items within a collection get ordered?

Another thing that helps is that you can put posts in a folder to indicate the category, we are using it for the courses we are doing atm (https://github.com/p2pu/jekyll-course-experiment). You can see the module content in folders called `community/_posts/`, `learning_posts/`, `content_posts/`, etc. That doesn't solve the issue of the date in the filename, but it helps a lot when it comes to organizing the different modules!
